lib/v2/gocn/topics.js

```diff
@@ -0,0 +1,26 @@
+const got = require('@/utils/got');
+
+module.exports = async (ctx) => {
+    const api_url = `https://gocn.vip/api/topic/list?currentPage=1&cate2Id=0&grade=new`;
+    const base_url = `https://gocn.vip/topics`;
+
+    const response = await got({
+        method: 'get',
+        url: api_url,
+        headers: {
+            Referer: base_url,
+        },
+    });
+    const data = response.data.data.list;
+
+    ctx.state.data = {
+        title: `GoCN社区文章`,
+        link: `https://gocn.vip`,
+        description: `获取GoCN站点最新文章`,
+        item: data.map((item) => ({
+            title: item.title,
+            description: item.summary,
+            link: `${base_url}/${item.id}`,
```

Comment:
```suggestion
            title: item.title,
            description: item.summary,
            link: `${base_url}/${item.id}`,
            pubDate: parseDate(item.ctime * 1000),
```
Please add pubDate
docs: https://docs.rsshub.app/joinus/pub-date.html

Reply:
my time is unix timestamp， how to do

Reply:
Pass the epoch time in ms to `parseDate` from '@/utils/parse-date', it will handle the reset

Reply:
![image](https://user-images.githubusercontent.com/39624029/154801685-4601597d-84cc-488a-af7a-b06f3d199882.png)
parse befor.

this is not error?

Reply:
code
![image](https://user-images.githubusercontent.com/39624029/154801797-1fece588-f341-45bc-a1db-940b3df122bb.png)

source
![image](https://user-images.githubusercontent.com/39624029/154801811-e6d755da-c95f-4c5a-abee-659963f9f956.png)